core/src/main/scala/akka/kafka/scaladsl/Consumer.scala

```diff
@@ -111,18 +111,27 @@ object Consumer {
    * Materialized value of the consumer `Source`.
    */
   trait Control {
+
+    /**
+     * Stop producing messages from the `Source`. This does not stop underlying kafka consumer
+     * and does not unsubscribe from any topics/partitions.
+     *
+     * Call [[#shutdown]] to close consumer
+     */
+    def stop(): Unit
```

Comment:
The stop itself (stop reading) is an async operation so I think it makes sense to return `Future[Done]` to signal that the stop operation has been done, i.e. no more messages will be emitted. Might be needed when chaining things.